src/hooks/useStableCallback.ts

```diff
@@ -8,16 +8,11 @@ type Callback = (...args: any[]) => any;
  */
 export const useStableCallback = (callback: Callback) => {
   const callbackRef = useRef<Callback>();
+  callbackRef.current = callback;
   const memoCallback = useCallback(
     // biome-ignore lint: to be addressed!
     (...args: any) => callbackRef.current && callbackRef.current(...args),
     []
   );
-  useEffect(() => {
-    callbackRef.current = callback;
-    return () => {
-      callbackRef.current = undefined;
```

Comment:
we need the effect at least for the clean up

Reply:
What exactly do you need to clean the callback up for?

Reply:
>       callbackRef.current = undefined;

freeing the ref object upon unmounting

Reply:
I understand. This is the first time I read anything about a need of cleaning up a ref object in a useEffect cleanup function. What advantages does it have?

Reply:
Ok, according to ChatGPT:

> Without a cleanup function, callbackRef.current is never cleared. This might lead to unexpected issues if there are asynchronous tasks that outlive the component lifecycle, as callbackRef may retain a reference to the function even after the component has unmounted. This can cause memory leaks or even runtime errors if asynchronous code tries to call callbackRef.current after unmounting.

I will modify the pull request, thank you.